### PR TITLE
Fix compilation errors on Windows

### PR DESCRIPTION
**Pull request by bellenot**
This should fix the errors LNK2019: unresolved external symbol "const ROOT::Experimental::RHistDrawable<1>::'vftable'"

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build with flags -DCMAKE_CXX_STANDARD=14 -Droot7=ON

File: hist/histdraw/v7/inc/ROOT/RHistDrawable.hxx [hunk 1]
```diff
@@ -83,71 +83,18 @@ public:
 
 };
 
-extern template class RHistDrawable<1>;
-extern template class RHistDrawable<2>;
-extern template class RHistDrawable<3>;
-
```
Comment (linev) [inline hist/histdraw/v7/inc/ROOT/RHistDrawable.hxx]: I have no idea why this code was here. 
@Axel-Naumann probably has better idea.
All other changes are ok for me.

Comment (bellenot) [reply hist/histdraw/v7/inc/ROOT/RHistDrawable.hxx]: OK, thanks @linev! I'll merge it once the builds are OK, and we'll see what @Axel-Naumann says...

**Comment by bellenot** [comment]
@phsft-bot build with flags -DCMAKE_CXX_STANDARD=14 -Droot7=ON

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/default with flags -DCMAKE_CXX_STANDARD=14 -Droot7=ON

**Pull request merged**